projects/netgrif-components/src/lib/navigation/breadcrumbs/breadcrumbs.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="breadcrumbs">
+    <div>
+        <span class="breadcrumbs-span" (click)="reset()">{{'toolbar.menu.home' | translate}}</span>
+        <span class="delimeter" *ngIf="getPath()?.length > 0">/</span>
+    </div>
+    <div *ngFor="let path of getPath();let i = index;">
```

Comment:
`getPath()` can return `undefined`. How does ngFor behave in this case? It would be better to change the `getPath()` method to always return an array

Reply:
done